contracts/FlowIDTableStaking.cdc

```diff
@@ -622,64 +593,68 @@ pub contract FlowIDTableStaking {
         }
     }
 
-    /// Admin resource that has the ability to remove insufficiently staked nodes
-    /// at the end of the staking auction, pay rewards to nodes at the end of an epoch, and move tokens between buckets
     pub resource Admin {
 
         /// Remove a node from the record
         pub fun removeNode(_ nodeID: String): @NodeRecord {
             let node <- FlowIDTableStaking.nodes.remove(key: nodeID)
                 ?? panic("Could not find a node with the specified ID")
 
-            FlowIDTableStaking.networkingAddressClaimed.remove(key: node.networkingAddress)
-            FlowIDTableStaking.networkingKeyClaimed.remove(key: node.networkingKey)
-            FlowIDTableStaking.stakingKeyClaimed.remove(key: node.stakingKey)
+            FlowIDTableStaking.updateClaimed(path: /storage/networkingAddressesClaimed, node.networkingAddress, claimed: false)
+            FlowIDTableStaking.updateClaimed(path: /storage/networkingKeysClaimed, node.networkingKey, claimed: false)
+            FlowIDTableStaking.updateClaimed(path: /storage/stakingKeysClaimed, node.stakingKey, claimed: false)
 
             return <-node
         }
 
         pub fun startStakingAuction() {
             pre {
-                !FlowIDTableStaking.stakingEnabled: "Cannot start staking auction if it is already in progress"
+                !FlowIDTableStaking.stakingEnabled(): "Cannot start staking auction if it is already in progress"
             }
-            FlowIDTableStaking.stakingEnabled = true
+            let stakingEnabled = FlowIDTableStaking.account.load<Bool>(from: /storage/stakingEnabled)
+
+            FlowIDTableStaking.account.save(true, to: /storage/stakingEnabled)
```

Comment:
Maybe refactor this and the similar code below in `endStakingAuction` into a function